### ISSUE TITLE
[Q]moving nodes

### ISSUE THREAD

**Issue by thob** [Kilian/fromscratch#23]
is there a shortcut for moving nodes (with all containing indented lines) up/down?

**Comment by Kilian** [comment]
There isn't one yet, good one. I'd like collapsed code to work the same as a single line. Ctrl up or down should work there. I'll look into it!

**Comment by nkantar** [comment]
Just for the record, on macOS `ctrl-up` activates Mission Control and `ctrl-down` shows the current application's windows. I believe Sublime Text and Atom use `ctrl-shift-up/down` for this functionality.

`cmd-up/down` could work well there, but will the `alt-up/down` analog on Windows/*nix clash with anything else?

**Comment by Kilian** [comment]
Yup, but for now all moving up/down acts on lines instead of on (collapsed) nodes.
